proto/internal/temporal/server/api/deployment/v1/message.proto

```diff
@@ -43,12 +43,14 @@ message DeploymentLocalState {
     map<string, TaskQueueFamilyData> task_queue_families = 2;
 
     bool is_current = 3;
-    google.protobuf.Timestamp last_became_current_time = 6;
-
     google.protobuf.Timestamp create_time = 4;
     map<string, temporal.api.common.v1.Payload> metadata = 5;
 
+    google.protobuf.Timestamp last_became_current_time = 6;
```

Comment:
please leave this where it was next to is_current. we should group things by related function. code is for humans to read, tag numbers are for the machines